src/libsync/creds/keychainchunk.h

```diff
@@ -39,7 +39,8 @@ static constexpr int MaxChunks = 10;
 /*
  * @brief: Abstract base class for KeychainChunk jobs.
  */
-class Job : public QObject {
+class Job : public QObject
```

Comment:
I think I'd mark it exported just to be on the safe side (although I think even Windows wouldn't bother us in that case since only the subclasses are used).

Reply:
Windows was happy with it, but sure, why not; let's play safe ^^